bigquery/cloud-client/sync_query_params.py

```diff
@@ -90,31 +94,118 @@ def sync_query_named_params(corpus, min_word_count):
     print_results(query_results)
 
 
+def sync_query_array_params(gender, states):
```

Comment:
General comment, no action necessary:

These samples seem .. cramped. It might help with readability if you put blank newlines between logical steps:

```python
client = bigquery.Client()

query = """
...
"""
query_results = client.run_sync_query(...)
query_results.use_legacy_sql = False

query_results.run()

print_results(query_results)
```